SYCL/Plugin/level_zero_imm_cmdlist_per_thread.cpp

```diff
@@ -0,0 +1,91 @@
+// REQUIRES: gpu, level_zero
+// TODO: ZE_DEBUG=4 produces no output on Windows. Enable when fixed.
```

Comment:
AFAIK, Windows now produces output when ZE_DEBUG=4, this issue was fixed.

Reply:
OK, let me enable windows and re-run.